vaadin-map-flow-parent/vaadin-map-flow/src/main/java/com/vaadin/flow/component/map/configuration/ViewExtent.java

```diff
@@ -0,0 +1,39 @@
+package com.vaadin.flow.component.map.configuration;
+
+/**
+ * Extent of the current view's state in EPSG:3857 format
+ */
+public class ViewExtent {
```

Comment:
```suggestion
/**
 * Defines an area within a map using min/max coordinates, which are by default in EPSG:3857 format
 */
public class Extent {
```
I think an extent is more of a general thing, and not only related to the currently visible area of the viewport.

Reply:
Refactored and made changes accordingly to method names.